.github/workflows/docker_publish.yml

```
@@ -238,51 +119,30 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        hdf5 : [
-          '',
-        ]
-        dagmc : [
-          '',
-          _dagmc
-        ]
-        pymoab : [
-          _pymoab,
-          ]
-        openmc : [
-          '',
-        ]
+        stage: [base_python, moab, dagmc, openmc]
+        hdf5: ['']
         include:
-          - hdf5: _hdf5-1_12_0
-            dagmc: _dagmc
-            pymoab: _pymoab
-            openmc: ''
-          - hdf5: ''
-            dagmc: _dagmc
-            pymoab: _pymoab
-            openmc: _openmc
-          - hdf5: ''
-            dagmc: ''
-            pymoab: ''
-            openmc: ''
-
+          - stage: dagmc
+            hdf5: /hdf5
 
-    name: "${{ github.repository_owner }}/ubuntu_20.04_py3${{ matrix.hdf5 }}${{ matrix.dagmc }}${{ matrix.pymoab }}${{ matrix.openmc }}: latest -> stable"
+    name: "${{ github.repository_owner }}/pyne/ubuntu_20.04_py3${{ matrix.hdf5 }}/${{ matrix.stage }}:refs_heads_multistage-docker-bk0: latest -> stable"
```

consistent naming with the basename and underscore
```suggestion
    name: "${{ env.DOCKER_IMAGE_BASENAME }}${{ matrix.hdf5 }}_${{ matrix.stage }}:refs_heads_multistage-docker-bk0: latest -> stable"
```